doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -23,23 +24,23 @@
 Traceability to Reference Model.
 
 <a name="2.3"></a>
-## 2.3 Open Stack Requirements.
-
-Proposals (WK): Change title of this section to 2.3 Architectural Requirements; propose the following structure of requirements:
+## 2.3 Requirements.
 
 <a name="2.3.1"></a>
 ### 2.3.1 General
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.gen.ost.01` | Open source | The Architecture shall/must xxx. |
-| `req.gen.ost.02` | Open source | The Architecture shall/must xxx. |
-| `req.gen.cnt.01` | Cloud nativeness | The Architecture shall/must xxx. |
-| `req.gen.cnt.02` | Cloud nativeness | The Architecture shall/must xxx. |
-| `req.gen.scl.01` | Scalability | The Architecture **shall** support policy driven auto-scaling. |
-| `req.gen.scl.02` | Scalability | The Architecture shall/must xxx. |
-| `req.gen.rsl.01` | Resiliency | The Architecture shall/must xxx. |
-| `req.gen.rsl.02` | Resiliency | The Architecture shall/must xxx. |
+| `req.gen.ost.01` | Open source | The Architecture **must** use OpenStack APIs.|
+| `req.gen.ost.02` | Open source | The Architecture **must** support dynamic request and configuration of virtual services (servers, networks, storage) through OpenStack APIs. |
+| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component.. |
+| `req.gen.cnt.02` | Cloud nativeness | The Architecture **should** consist of service components implemented as microservices that are individually dynamically scalable. |
+| `req.gen.cnt.03` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where the state is required it **must** be kept external to the component. |
+| `req.gen.scl.01` | Scalability | The Architecture **should** support policy driven auto-scaling. |
+| `req.gen.rsl.01` | Resiliency | The Architecture **must** support resilient OpenStack components that are required for the continued operation of running workloads. |
+| `req.gen.rsl.02` | Resiliency | The Architecture **should** support resilient OpenStack service components that are are not subject to `req.gen.rsl.01`. |
+| `req.gen.avl.01` | Availability | The Architecture **must** provide High Availability for OpenStack components |
```

Comment:
Instead "The Architecture **must** provide High Availability for OpenStack components"
I would propose: "The Architecture **must** provide High Availability for OpenStack APIs"

Reply:
Why only APIs?  In real implementation every component of an Architecture (inn this case OS) must be HA.  The API itself will not do anything if a component which stands behind it is not available.  Usually these components are deployed in clusters of 3 (quorum), but we do not want to specify the solution here.

Reply:
You are right -- I implied successful execution of API calls within the defined metrics. Not sure i all OpenStack components need to be HA for this purpose.

Reply:
I go with @walterkozlowski  comment